SU-SWS/stanford_profile
# 8.2.6 hotfix: added experimental label to basic page type field and view

# READY FOR REVIEW

# Summary
- Added "experimental" label to basic page type field and view

# Needed By (Date)
- ASAP

# Urgency
- needed for 2.1.0

# Steps to Test

1. checkout branch
2. drush cim
3. check the node creation page to make sure that the "(experimental)" text is added to the basic page type select menu
4. create a list paragraph, and make sure the view selection says "(experimental)" next to Basic Pages

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
